src/avfoundation.cs

```diff
@@ -9183,6 +9373,7 @@ interface AVCaptureResolvedPhotoSettings
 		[Export ("flashEnabled")]
 		bool IsFlashEnabled { [Bind ("isFlashEnabled")] get; }
 
+		[Deprecated (PlatformName.iOS, 13,0, message: "Use 'AVCaptureResolvedPhotoSettings.PhotoProcessingTimeRange' instead.")]
```

Comment:
add `[NoMac]` (no point in adding deprecated API) unless the new API is not available on macOS